sdk/scheduler/src/main/java/org/apache/mesos/specification/yaml/RawContainer.java

```diff
@@ -16,4 +20,13 @@ public String getImageName() {
     public void setImageName(String imageName) {
         this.imageName = imageName;
     }
+
+    public LinkedHashMap<String, RawRLimit> getRLimits() {
+        return rlimits;
+    }
+
+    @JsonProperty("rlimits")
```

Comment:
The `set*()` methods could be removed and replaced by a `@JsonCreator`-style constructor. Then the `imageName`/`rlimits` members could be marked `final`.